monome/teletype
# Add new OPs: I2M.MUTE / I2M.SOLO

#### What does this PR do?

Adds two new OPs for I2C2MIDI:
- `I2M.MUTE x`: Get mute state / Set mute state of current MIDI channel to `x` (0..1) 
- `I2M.SOLO x`: Get solo state / Set solo state of current MIDI channel to `x` (0..1)

… as well as their channel-specific counterparts:
- `I2M.MUTE# ch x`: Get mute state / Set mute state of MIDI channel `ch` to `x` (0..1)
- `I2M.SOLO# ch x`: Get solo state / Set solo state of MIDI channel `ch` to `x` (0..1)

#### Provide links to any related discussion on [lines](https://llllllll.co/).
Has been discussed on the TT discord.

#### How should this be manually tested?
With I2C2MIDI connected …
1. Send two MIDI notes on channel 1:
```
#1 
I2M.CH 1
I2M.N 60 127
DEL 500: I2M.N 60 127
```
2. Add a line to mute the current channel. This should now only send one MIDI note:
```
I2M.CH 1
I2M.N 60 127
I2M.MUTE 1
DEL 500: I2M.N 60 127
```
Apply similar techniques for I2M.MUTE#, I2M.SOLO and I2M.SOLO#.

#### Any background context you want to provide?
—

#### If the related Github issues aren't referenced in your commits, please link to them here.
—

#### I have,
  → Did not add anything, I2C2MIDI OPs are already mentioned there
  → as discussed with @scanner-darkly I skipped this step
